test/sqllogictest/chbench.slt

```diff
@@ -693,80 +663,65 @@ AND i_id = ol_i_id
+Each group of RelationExprs is referred by id eg %4
+Columns are referred to by position eg #4
+RelationExprs may be followed by additional annotations on lines starting with | |
+
+0 =
+| Get materialize.public.item (u24)    
+| ArrangeBy (#0)    
+
+1 =
+| Get materialize.public.supplier (u34)    
+| ArrangeBy (#0) (#3)    
+
+2 =
+| Get materialize.public.stock (u26)    
+| ArrangeBy (#0) (#0, #1) (#17)    
+
+3 =
+| Get materialize.public.orderline (u19)    
+| ArrangeBy (#2, #1, #0) (#5, #4)    
+
+4 =
+| Get materialize.public.order (u16)    
+| ArrangeBy (#0, #1, #2) (#2, #1, #3)    
+
+5 =
+| Get materialize.public.customer (u6)    
+| ArrangeBy (#0, #1, #2) (#21)    
+
+6 =
+| Get materialize.public.nation (u31)    
+| ArrangeBy (#0) (#2)    
+
+7 =
+| Get materialize.public.nation (u31)    
+| ArrangeBy (#0)    
+
+8 =
+| Get materialize.public.region (u37)    
+| ArrangeBy (#0)    
+
+9 =
+| Join %0 %1 %2 %3 %4 %5 %6 %7 %8 (= %0#0 %2#0 %3#4) (= %1#0 %2#17) (= %1#3 %7#0) (= %2#1 %3#5) (= %3#0 %4#0) (= %3#1 %4#1 %5#1) (= %3#2 %4#2 %5#2) (= %4#3 %5#0) (= %5#21 %6#0) (= %6#2 %8#0)    
+| | implementation = DeltaQuery([[(2, [Column(0)]), (1, [Column(0)]), (7, [Column(0)]), (3, [Column(5), Column(4)]), (4, [Column(0), Column(1), Column(2)]), (5, [Column(0), Column(1), Column(2)]), (6, [Column(0)]), (8, [Column(0)])], [(7, [Column(0)]), (2, [Column(17)]), (0, [Column(0)]), (3, [Column(5), Column(4)]), (4, [Column(0), Column(1), Column(2)]), (5, [Column(0), Column(1), Column(2)]), (6, [Column(0)]), (8, [Column(0)])], [(0, [Column(0)]), (1, [Column(0)]), (7, [Column(0)]), (3, [Column(5), Column(4)]), (4, [Column(0), Column(1), Column(2)]), (5, [Column(0), Column(1), Column(2)]), (6, [Column(0)]), (8, [Column(0)])], [(4, [Column(0), Column(1), Column(2)]), (5, [Column(0), Column(1), Column(2)]), (2, [Column(0), Column(1)]), (0, [Column(0)]), (1, [Column(0)]), (6, [Column(0)]), (7, [Column(0)]), (8, [Column(0)])], [(5, [Column(0), Column(1), Column(2)]), (6, [Column(0)]), (8, [Column(0)]), (3, [Column(2), Column(1), Column(0)]), (2, [Column(0), Column(1)]), (0, [Column(0)]), (1, [Column(0)]), (7, [Column(0)])], [(6, [Column(0)]), (8, [Column(0)]), (4, [Column(2), Column(1), Column(3)]), (3, [Column(2), Column(1), Column(0)]), (2, [Column(0), Column(1)]), (0, [Column(0)]), (1, [Column(0)]), (7, [Column(0)])], [(8, [Column(0)]), (5, [Column(21)]), (4, [Column(2), Column(1), Column(3)]), (3, [Column(2), Column(1), Column(0)]), (2, [Column(0), Column(1)]), (0, [Column(0)]), (1, [Column(0)]), (7, [Column(0)])], [(1, [Column(3)]), (2, [Column(17)]), (0, [Column(0)]), (3, [Column(5), Column(4)]), (4, [Column(0), Column(1), Column(2)]), (5, [Column(0), Column(1), Column(2)]), (6, [Column(0)]), (8, [Column(0)])], [(6, [Column(2)]), (5, [Column(21)]), (4, [Column(2), Column(1), Column(3)]), (3, [Column(2), Column(1), Column(0)]), (2, [Column(0), Column(1)]), (0, [Column(0)]), (1, [Column(0)]), (7, [Column(0)])]])    
```

Comment:
Can this be made more concise?

Reply:
We might want to try it out as a multi-line thing, but it may also be appropriate to not present it (by default at least); the information is quadratic in the number of inputs, and there isn't any particular reason it should compress down well.

Reply:
The `DeltaQuery` bit onwards is actually a TODO that I forgot :sheep: